contracts/MerkleDelegation.sol

```diff
@@ -50,19 +51,14 @@ contract MerkleDelegation is Ownable, Pausable {
         require(delegator != address(0), "DR: delegator must be non-zero");
```

Comment:
question: what does `DR` stand for?

The contract name is MerkleDelegation, that would abbreviate to `MD` 🤔

Reply:
it was DelegationRegistry, to be updated...

Reply:
Snapshot uses block numbers, what do you mean by using a "snapshot" approach?

Reply:
I was reviewing yesterday the snapshot data from BIT contract and it unrelated with the voting power, I think the only use the stored snapshot block numbers to check that the Proposals are real. Correct me if I am wrong.

Reply:
I am reading the Snapshots implementation on the BIT token and is very similar to what I am doing, I was just doing what I recall from the implementation but used the name *checkpoints*.